SU2_CFD/src/output/COutput.cpp

```diff
@@ -191,7 +191,7 @@ void COutput::SetHistoryOutput(CGeometry *geometry,
                                   unsigned long InnerIter) {
 
   curTimeIter  = TimeIter;
-  curAbsTimeIter = TimeIter - config->GetRestart_Iter();
+  curAbsTimeIter = max(TimeIter, config->GetStartWindowIteration()) - config->GetStartWindowIteration();
```

Comment:
Is this variable only used for the volume time averages?

Reply:
Right，curAbsTimeIter is only used in void COutput::SetAvgVolumeOutputValue.
![var_in_file](https://github.com/su2code/SU2/assets/58663453/d9959d87-f475-4dd3-9643-c24b6a67cdcc)